lib/metrics.js

```diff
@@ -47,10 +47,12 @@ export class Metrics {
     this.moduleMetrics.set(moduleName, metrics)
     const mergedMetrics = {
       totalJobsCompleted: 0,
-      rewardsScheduledForAddress: '0'
+      rewardsScheduledForAddress: 0n
     }
     for (const [, metrics] of this.moduleMetrics) {
       mergedMetrics.totalJobsCompleted += metrics.totalJobsCompleted
+      mergedMetrics.rewardsScheduledForAddress +=
+        metrics.rewardsScheduledForAddress
```

Comment:
Composing rewards is tricky.
- If you run both Station Core and Station Desktop, this metric will show rewards for both apps combined together.
- If let's say Bacalhau starts paying rewards, I think it's safe to assume they will use a different smart contract, therefore the Bacalhau module will report different scheduled rewards than SPARK, and thus it's safe to sum these amounts as you are doing here.

I think it will be good to point out this complexity by adding a code comment.

WDYT?

Reply:
> If you run both Station Core and Station Desktop, this metric will show rewards for both apps combined together.

This is true, but independent of merging module metrics here, right?

> If let's say Bacalhau starts paying rewards, I think it's safe to assume they will use a different smart contract, therefore the Bacalhau module will report different scheduled rewards than SPARK, and thus it's safe to sum these amounts as you are doing here.

I'll add a code comment 👍

Reply:
> > If you run both Station Core and Station Desktop, this metric will show rewards for both apps combined together.
> 
> This is true, but independent of merging module metrics here, right?

Yes, I think so.